### PR TITLE
**Fix**: Model Confirm Styling

### PR DESCRIPTION
**Pull request by ericadamski**
# Summary

As mentioned in #698 the modal styling had been previously broken. This PR serves to patch it. Below are some screenshots of the patched renders of the model on `styleguidist` and `dev` respectfully.

screen shot 2018-09-05 at 8 05 54 am
screenshot_2018-09-05 screenshot

# Related issue

#698 

# To be tested

Me
- [x] No error or warning in the console on `localhost:6060`

Tester 1

- [x] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

**Comment by ericadamski** [comment]
@TejasQ this is what happens on CI 😢 
screen shot 2018-09-05 at 8 27 13 am

**Comment by TejasQ** [comment]
Thanks, Eric. It's strange though because it's _in_ the build config.

image

**Comment by ericadamski** [comment]
That is odd. 🤔

**Comment by TejasQ** [comment]
@ericadamski alright, I think I've fixed it on master. Mind rebasing on more time for me?

**Comment by ericadamski** [comment]
🎉 🎉 🎉  Nice fix @TejasQ !
screenshot_2018-09-05 fix model confirm styling by ericadamski pull request 723 contiamo operational-ui

**Comment by peterszerzo** [comment]
@ericadamski looking great! The absolute positioning will cause the actions to cover the text like in the screenshot below. Now since this is a confirm component with ideally not much text, it's certainly not worth breaking our neck to support this, but just in case anyone abuses the space, I think it would be safer if the actions stayed in the document flow and would only be reachable by scrolling down in case the content is too tall.

screen shot 2018-09-05 at 3 01 04 pm

**Comment by fabien0102** [comment]
@peterszerzo and me are so sync! I'm afraid ^^

**Comment by ericadamski** [comment]
I am not super thrilled with this fix. I don't have a great grasp of the organization and hierarchy of the library so making this change to the `Card` generic component feels a little odd. Here are some screenshots of the results and one side effect of the change to `Card`

Fixed with short message:
screen shot 2018-09-05 at 9 34 35 pm
Fixed with longer message:
screen shot 2018-09-05 at 9 34 43 pm

Original before fix:
screen shot 2018-09-05 at 9 44 11 pm

Issue after fix:
screen shot 2018-09-05 at 9 44 26 pm

Possible solution?
screen shot 2018-09-05 at 9 44 41 pm

**Comment by ericadamski** [comment]
:100: I prefer that for sure!

**Comment by ericadamski** [comment]
@TejasQ how important is the `min-height` on the `ControlledModal`?

https://github.com/contiamo/operational-ui/blob/ceacae831399c56ef94912f08700a1673ddd2559/src/Internals/ControlledModal.tsx#L34

**Comment by ericadamski** [comment]
This is without the `min-height`
screen shot 2018-09-06 at 6 58 21 am

**Comment by fabien0102** [comment]
To add more context, the `min-height` reflect our design specs.

The screenshot for our internal zeplin:
image

File: src/Internals/Confirm.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import * as React from "react"
-import styled from "../utils/styled"
-
 import Button, { ButtonProps } from "../Button/Button"
+import styled from "../utils/styled"
 import ControlledModal from "./ControlledModal"
 
 const Actions = styled("div")({
+  marginTop: 20,
```
Comment (TejasQ) [inline src/Internals/Confirm.tsx]: Actually, there's a `theme` variable here.

If this object literal is instead a function, it receives props as its first argument. Then, you can use `props.theme.space.element` here in order to be really adherent to our designs. 

Great work, Eric!

**Pull request merged**